arm/examples/check/check.go

```diff
@@ -31,7 +33,9 @@ func byInspecting() autorest.RespondDecorator {
 	}
 }
 
-func checkName(name string) {
+func main() {
+	name := fmt.Sprintf("testname%s", randomString(5))
```

Comment:
while you're at it, maybe just hardcode a string here and get rid of random thing. it'll be more clear to understand.